<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>期权类型</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelStartPrice">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>320</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>期权执行价格（元）</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnUp">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#4d85ff;color:white;</string>
   </property>
   <property name="text">
    <string>欧式看涨期权</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDown">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>20</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>欧式看跌期权</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>161</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>资产初始价格（元）</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>期权合约期限（月）</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>无风险利率（%）</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>年化波动率（%）</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>期权执行价格（元）</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>70</y>
     <width>291</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QDoubleSpinBox" name="input1">
      <property name="minimum">
       <double>1.000000000000000</double>
      </property>
      <property name="maximum">
       <double>10.000000000000000</double>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="input2">
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="minimum">
       <double>1.000000000000000</double>
      </property>
      <property name="maximum">
       <double>12.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.500000000000000</double>
      </property>
      <property name="value">
       <double>3.000000000000000</double>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="input3">
      <property name="minimum">
       <double>0.500000000000000</double>
      </property>
      <property name="maximum">
       <double>3.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
      <property name="value">
       <double>1.500000000000000</double>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="input4">
      <property name="minimum">
       <double>5.000000000000000</double>
      </property>
      <property name="maximum">
       <double>50.000000000000000</double>
      </property>
      <property name="value">
       <double>5.000000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="startCalcBtn">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>360</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>执行计算</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>150</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>范围：1 - 12</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>210</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>范围：0.5 - 3</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>260</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>范围：5 - 10</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="calcProgressBar">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>180</y>
     <width>531</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximum">
    <number>0</number>
   </property>
   <property name="value">
    <number>-1</number>
   </property>
   <property name="format">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
